config/routes.rb

```diff
@@ -29,6 +29,12 @@
     end
   end
 
+  resources :finishers, only: [] do
```

Comment:
I think we want this under the manage namespace

Reply:
Thanks for this feedback! I've moved it under manage projects now.